django/thunderstore/core/settings.py

```diff
@@ -279,9 +280,12 @@ def load_db_certs():
 
 # Session cookie used by React components during Django React transition
 SESSION_COOKIE_HTTPONLY = False
+SESSION_COOKIE_DOMAIN = None if \
```

Comment:
I am fairly sure you can simply do `env.str("SESSION_COOKIE_DOMAIN")` if the default is `None`. Not 100% sure but 90% sure. It'll work if `django-environ` doesn't convert the default value to string automatically, which I think it doesn't

Reply:
Moreover, even if it would get converted to a string, we can default to `""` as it is in the current form and write down
```
SESSION_COOKIE_DOMAIN = env.str("SESSION_COOKIE_DOMAIN") or None
```

Reply:
Well that might work, hold on

Reply:
`SESSION_COOKIE_DOMAIN = env.str("SESSION_COOKIE_DOMAIN") or None`
This works fine, but it doesn't check if the value has been assigned as "None". Might be a problem for someone, but also leaving the "None string check" out, makes the code a bit cleaner.

Reply:
If someone assigns `"None"` to it, they're configuring the system wrong and it's allowed to fail, so no need to try handling it